### ISSUE TITLE
Is this repo/project abandoned?

### ISSUE THREAD

**Issue by SublimeAnanas** [codebicycle/videospeed#199]
I was going to start a new issue regarding the extension and reddit's video player, but considering the extension hasn't been updated for over a year I'm wondering if the extension has been abandoned by the dev. Apologies if this isn't the right place to ask, I'm new to github, but I'm essentially wondering if I should switch my Firefox extension from this version to a forked, more up-to-date one.

**Comment by gediminasel** [comment]
Yes, I think it is abandoned. I created my own fork and merged [igrigorik/videospeed](https://github.com/igrigorik/videospeed) updates: [gediminasel/videospeed-firefox](https://github.com/gediminasel/videospeed-firefox).
Mozilla Add-ons: https://addons.mozilla.org/en-US/firefox/addon/video-speed-controller-new/
From a few initial tests, it seems to solve #194 #140 #173 and similar issues.

**Comment by SublimeAnanas** [comment]
@gediminasel thank you, I installed your addon. are you aware of any other popular fork of this project that is available in the firefox/chrome add-on/extension store?

**Comment by gediminasel** [comment]
Hi, the original [igrigorik/videospeed](https://github.com/igrigorik/videospeed) extension is for Chrome and should work just fine. I'm not aware of any Firefox port, that's why I created mine.

**Comment by Dreamer8** [comment]
> 

Hello, could you check also this issue, please?

https://github.com/codebicycle/videospeed/issues/180

I miss a button to toggle actual and original speed.

Currently I have to click 5x to change to original speed and then 5x to switch it back, with that button it would be just 1 click.

**Comment by gediminasel** [comment]
Hi @Dreamer8,

> Hello, could you check also this issue, please?

[igrigorik/videospeed](https://github.com/igrigorik/videospeed) behaves the same on #180, report the bug there. 

> I miss a button to toggle actual and original speed.

Try keyboard shortcuts (R - original, G - preferred, see more in extension options).

**Comment by Dreamer8** [comment]
Hi @gediminasel,

thanks for the reply, I don't like the keyboard shortcuts, because I'm using mostly mouse, especially when watching videos.

Keyboard shortcuts could cause also problems, conflicts...

I'm used to the toggle button, I'm using edited skin also in my video player just for this purpose.

Alternatively it could be 2 buttons - original and last used speed, or original and custom speed.

Another option could be the main addon button - 1-click to toggle original/last used, or original/custom speed, I'm using lot of 1-click addons already. Left click could be toggle and right click to set the speed with more options.

**Comment by gediminasel** [comment]
Hi, please request new features [igrigorik/videospeed](https://github.com/igrigorik/videospeed/issues). If they add those, I'll port them to firefox extension. You can also take a look at this: [magnus-ISU/videospeed](https://github.com/magnus-ISU/videospeed/issues). Might be better than my fork.

**Comment by Dreamer8** [comment]
Done, and I found a workaround too, not perfect, still need 2 buttons, but at least less clicks.

I changed the intervals to 0.25, so now I can use the existing buttons, just 2 clicks instead of 5.

**Comment by Dreamer8** [comment]
Another update, the hotkey "/" works good with the experimental feature I found "Disable website key bindings", so at least it works the same as in my video player with keyboard shortcuts.

**Comment by SoPat712** [comment]
I've been working on my fork actively here!

https://github.com/SoPat712/Speeder

I'm happily accepting any bug/feature requests you have, so let me know if you end up trying it out.
